src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PackageRestore/PackageRestoreDataSource.cs

```diff
@@ -63,9 +63,13 @@ internal class PackageRestoreDataSource : ChainedProjectValueDataSourceBase<Rest
         private readonly IFileSystem _fileSystem;
         private readonly IProjectLogger _logger;
         private readonly IProjectDependentFileChangeNotificationService _projectDependentFileChangeNotificationService;
-
+        private readonly IVsSolutionRestoreService4 _solutionRestoreService4;
         private byte[]? _latestHash;
         private bool _enabled;
+        private bool _pendingNomination = true;
```

Comment:
Consider renaming as `_hasPendingNomination`, making it clear that this is a boolean and not some object representing the pending nomination's data.